addon-test-support/scroll-into-view-mock.js

```diff
@@ -0,0 +1,31 @@
+import sinon from 'sinon';
```

Comment:
We will need to mark `sinon` as a peerDependency and remember to document it as a requirement for using this test helper. We don't want to make it a true dependency, because that would require it for production builds.